entity-services/src/main/xdmp/entity-services/entity-services-impl.xqy

```diff
@@ -656,10 +656,11 @@ declare function esi:database-properties-generate(
         for $word-lexicon-property in json:array-values($word-lexicon-properties)
         let $property := $entity-type=>map:get("properties")=>map:get($word-lexicon-property)
         let $collation := head( (map:get($property, "collation"), "http://marklogic.com/collation/en") )
+        let $namespace-uri := head( ($entity-type=>map:get("namespace"), "") )
         let $wl-map := json:object()
             =>map:with("collation", $collation)
             =>map:with("localname", $word-lexicon-property)
-            =>map:with("namespace-uri", "")
+            =>map:with("namespace-uri", $namespace-uri)
```

Comment:
I just ran unit and functional tests to see $namespace-uri instead of "" for model with no namespaces. Strangely enough it all works OK now after I fetched your latest change.